tests/subsys/net/lib/nrf_cloud/cloud/src/main.c

```diff
@@ -205,7 +205,7 @@ static void run_before(void *fixture)
 	RESET_FAKE(nct_dc_stream);
 	RESET_FAKE(nct_dc_send);
 	RESET_FAKE(nct_dc_bulk_send);
-	RESET_FAKE(nrf_cloud_device_status_encode);
+	RESET_FAKE(nrf_cloud_device_status_shadow_encode);
```

Comment:
You also need to change the 4 other locations in this file that refer to `nrf_cloud_device_status_encode_fake` to be `nrf_cloud_device_status_shadow_encode_fake`.